src/index.js

```diff
@@ -199,9 +199,9 @@ class HCaptcha extends React.Component {
     execute () {
```

Comment:
Sorry I mean to add the ability to turn async on if the user wanted to by passing it as a parameter.
```js
execute(config={}) {
    // logic
    return hcaptcha.execute(captchaId, config);
}
```